packages/next/client/image.tsx

```diff
@@ -5,10 +5,11 @@ const loaders = new Map<LoaderKey, (props: LoaderProps) => string>([
   ['imgix', imgixLoader],
   ['cloudinary', cloudinaryLoader],
   ['akamai', akamaiLoader],
+  ['bigCommerce', bigCommerceLoader],
```

Comment:
`bigCommerce` looks a little strange... perhaps we can just lowercase the entire word

`bigcommerceLoader`

Reply:
Updated 👍